### ISSUE TITLE
Mobile

### ISSUE THREAD

**Issue by Dmi3yy** [WorkForFood/MODxPress#9]
НА iphone6 форма входа:

логотип обрезало видно только хвостик и надпись MODX
кнопка войти печально выглядит
так же не стоит viewport, открывает на телефоне в полный размер

**Comment by WorkForFood** [comment]
Да, с яблочными телефонами есть проблемы. Допилю к следующему коммиту. Все мета тоже добавлю.

**Comment by WorkForFood** [comment]
Кнопку поправил, мета с вьюпортами добавил.
Форму авторизации сейчас допилю.
